src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -158,8 +159,8 @@ describe('Exam assessment', () => {
             if (dayjs().isBefore(examEnd)) {
                 cy.wait(examEnd.diff(dayjs(), 'ms'));
             }
-            cy.login(admin, `/course-management/${course.id}/exams/${exam.id}/student-exams`);
-            studentExamManagement.clickEvaluateQuizzes().its('response.statusCode').should('eq', 200);
+            cy.login(admin, `/course-management/${course.id}/exams/${exam.id}/assessment-dashboard`);
+            assessmentDashboard.clickEvaluateQuizzes();
```

Comment:
Should we expect the 200 status here as well?

Reply:
Yes.